### DISCUSSION TITLE
"finished getting page [#]. starting page [#]"

### DISCUSSION THREAD

**Discussion by katianahancock** [nianeyna/ao3downloader#104] [Q&A]
Hi! First of all, I want to thank you SO much for writing and maintaining this script - I have 3k+ AO3 bookmarks and this is an absolute lifesaver for me.

Second, I'm encountering an interesting issue. (I swear I've followed all the troubleshooting steps before posting!) At a certain point (**EDIT** - after testing several more times, it stops at 13%!) in the process, when logged in to my AO3 account, the script goes into an endless loop of getting "pages". This definitely only happens when I'm logged in (tested several times - works just fine logged out) and cycles endlessly. I'd do the whole thing logged out to get around the issue, but a lot of fics these days are locked and I'm coming up with less than half. Any ideas as to what's going on here? It sounds potentially like your first bullet point under Known Issues - the cycling infinitely one - but I don't know enough about what that looks like to say for sure that it's the same thing, especially when being logged in appears to be the problem. It definitely isn't downloading anything else while it's cycling through its page-getting - I've monitored the number of files in the folder and it doesn't change after that point.

Any thoughts? Thank you so much again for all of your hard work!

**Comment by nianeyna** [comment]
Thanks for reporting this! Would you mind emailing me (nianeyna at gmail) your log.jsonl file?
